paper/tex/results.tex

```diff
@@ -1,386 +1,146 @@
-%\textsc{SkyNet} 	& $0.0483$ & $385.830$ &  $2119.4$ &  $2110.3$ \\
-%\textsc{TPZ} 		& $0.0126$ &   $9.483$ &   $110.7$ &   $110.6$ \\
-%%\textsc{Frankenz}	& $-$ & $-$ & $-$ & $-$ \\
-%\hline
-%\end{tabular}
-%\end{table*}
+The values of the CDE loss statistic of individual \pzpdf\ accuracy are provided in Table~\ref{tab:cdeloss}.
+It is worth noting that strong performance on the CDE loss should imply strong performance on the other metrics, though the inverse is not necessarily true.
+Thus the CDE loss is the most effective metric for generic science cases.
 
 \begin{table}  %%% DATA TABLE %%%
 \centering
-\caption{CDE loss statistic for each photo-$z$ code.} \label{tab:cdeloss}
+\caption{CDE loss statistic of the individual \pzpdf s for each code.
+A lower value of the CDE loss indicates more accurate individual \pzpdf s, with \cmnn\ and \flexzboost\ performing best under this metric.}
+\label{tab:cdeloss}
 \begin{tabular}{lr}
 \hline
-\bf Photo-$z$ Code & \bf CDE Loss \\
+Photo-$z$ Code & CDE Loss \\
 \hline
-\textsc{ANNz2} 		& $-6.88$ \\
-\textsc{BPZ} 		& $-7.82$ \\
+\annz 	    & $-6.88$ \\
+\bpz 		    & $-7.82$ \\
 %\textsc{Delight} 	& $-4.06$ \\
-\textsc{Delight}    & $-8.33$\\
+\delight    & $-8.33$\\
 %\textsc{EAZY} 		& $-7.97$ \\
-\textsc{EAZY}       & $-7.07$ \\
+\eazy       & $-7.07$ \\
 %\textsc{FlexZBoost} & $-11.51$ \\
-\textsc{FlexZBoost} & $-10.60$\\
-\textsc{GPz} 		& $-9.93$ \\
-\textsc{LePhare} 	& $-1.66$ \\
-\textsc{METAPhoR} 	& $-6.28$ \\
+\flexzboost & $-10.60$\\
+\gpz		    & $-9.93$ \\
+\lephare 	  & $-1.66$ \\
+\metaphor 	& $-6.28$ \\
 %\textsc{CMNN} 		& $-$ \\
-\textsc{CMNN}         & $-10.43$ \\
-\textsc{SkyNet} 	& $-7.89$ \\
-\textsc{TPZ} 		& $-9.55$ \\
+\cmnn       & $-10.43$ \\
+\skynet 	  & $-7.89$ \\
+\tpz 		    & $-9.55$ \\
 \hline
-\trainz		& $-0.83$ \\
+\trainz		  & $-0.83$ \\
 %\textsc{Frankenz}	& $-$  \\
 %\hline
 \end{tabular}
 \end{table}
 
-%\begin{table}
-%\setlength{\tabcolsep}{2pt}
-%\begin{center}
-%\caption{Root-Mean-Square-Error (RMSE) statistics for the twelve photo-z codes for both PIT and $\hat{N}(z)$ distributions.}\label{tab:rmse}
-%\begin{tabular}{lcc}
-%\hline
-%\hline
-%Root-Mean-Square-Error \\
-%(RMSE) statistics \\
-%\hline
-%Photo-z Code & PIT RMSE & N(z) RMSE\\
-%\hline
-%\textsc{ANNz2}      & 0.019 & 0.0054\\
-%\textsc{BPZ}        & 0.032 & 0.0050\\
-%\textsc{Delight}    & 0.111 & 0.0056\\
-%\textsc{EAZY}       & 0.054 & 0.0102\\
-%\textsc{FlexZBoost} & 0.021 & 0.0022\\
-%\textsc{GPz}        & 0.027 & 0.0042\\
-%\textsc{LePhare}    & 0.028 & 0.0062\\
-%\textsc{METAPhoR}   & 0.064 & 0.0081\\
-%\textsc{CMNN}         & 0.108 & 0.0009\\
-%\textsc{Skynet}     & 0.054 & 0.0144\\
-%\textsc{TPZ}        & 0.082 & 0.0031\\
-%\hline
-%\trainz		& 0.0025 & 0.0013\\
-%\end{tabular}
-%\end{center}
-%\end{table}
+This metric is the only one that can appropriately penalize \trainz\ and indicates strong performance for \cmnn\ and \flexzboost, the latter of which is optimized for this metric.
```

Comment:
This seems too brief, given that it's one of the more important points in the paper, can we beef this back up a bit with another sentence or two?

Reply:
I split it up and added boldface to the most important part.  Is that any better?